test/planned_subvol_test.rb

```diff
@@ -207,4 +208,122 @@
     end
   end
 
+  context ".read_from_xml" do
```

Comment:
Rspec lingo calls those things "examples". Surely there must be a reason for that?

Yes, of course I could have covered the entire fallback list verbatim. But then the tests would break every single time somebody requests yet another subvolume or requests a subvolume to be NoCOW. This would be unmaintainable very quickly.

But I forgot, we are not doing this to do any useful testing, we are just blindly fulfilling metrics. That does not strike me as very useful.